crates/examples/src/readobj/elf.rs

```diff
@@ -162,6 +162,8 @@ fn print_program_headers<Elf: FileHeader>(
                 //PT_GNU_STACK =>
                 //PT_GNU_RELRO =>
                 //PT_GNU_PROPERTY =>
+                //GNU_PROPERTY_AARCH64_FEATURE_1_AND =>
+                //GNU_PROPERTY_X86_FEATURE_1_AND =>
```

Comment:
I don't think this should be added here. These are not segment types.

Reply:
OK, thanks